src/traffic-collector/azext_traffic_collector/aaz/latest/network_function/traffic_collector/_list.py

```diff
@@ -0,0 +1,576 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "network-function traffic-collector list",
+)
+class List(AAZCommand):
+    """Returns list of Azure Traffic Collectors in selected subscription
```

Comment:
It seems this command contains both by a subscription and by a resource group
```suggestion
    """Returns list of Azure Traffic Collectors
```